### PR TITLE
fix(component): icon alignment and autofill input fixes

### PR DESCRIPTION
**Pull request by chanceaclark**
## What
Inputs with `Icons` are not perfectly aligned and have extra margins / paddings. This PR is an attempt to resolve that, alongside ensuring the whole field upon autocomplete is filled.

## Screenshot
Screen Shot 2020-01-15 at 2 41 20 PM

### CODE REVIEW

File: packages/big-design/src/components/Input/styled.tsx [hunk 1]
```diff
@@ -68,47 +69,92 @@ export const StyledInput = styled.input<InputProps>`
     font-size: ${({ theme }) => theme.typography.fontSize.medium};
   }
 
-  ${props =>
-    props.iconRight &&
+  &:-webkit-autofill {
+    background-color: ${({ theme }) => theme.colors.primary10} !important;
```
Comment (deini) [inline packages/big-design/src/components/Input/styled.tsx]: Can we add a comment on why we are using `!important`?

File: packages/big-design/src/components/Input/styled.tsx [hunk 2]
```diff
@@ -68,47 +69,92 @@ export const StyledInput = styled.input<InputProps>`
...
+    -webkit-box-shadow: 0 0 0px 1000px ${({ theme }) => theme.colors.primary10} inset;
```
Comment (deini) [inline packages/big-design/src/components/Input/styled.tsx]: This throws a warning on `ts-styled-plugin` 🤔 is there a way to ignore it?

Comment (chanceaclark) [reply packages/big-design/src/components/Input/styled.tsx]: I'll 👀 to see if there's a way

Comment (chanceaclark) [reply packages/big-design/src/components/Input/styled.tsx]: Unfortunately there isn't a way to disable it on a per-line basis:
https://github.com/Microsoft/typescript-styled-plugin/issues/70#issuecomment-420457682

We could disable linting validation entirely, but I assume we wouldn't want that.

**Pull request merged**